cni/network/stateless/main.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2017 Microsoft. All rights reserved.
+// MIT License
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-container-networking/cni"
+	zapLog "github.com/Azure/azure-container-networking/cni/log"
+	"github.com/Azure/azure-container-networking/cni/network"
+	"github.com/Azure/azure-container-networking/common"
+	"github.com/Azure/azure-container-networking/log"
+	"github.com/Azure/azure-container-networking/nns"
+	"github.com/Azure/azure-container-networking/platform"
+	"github.com/Azure/azure-container-networking/telemetry"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+)
+
+var logger = zapLog.InitZapLogCNI(name, "azure-vnet.log")
```

Comment:
this needs to be changed..check with paul yu

Reply:
yeah, I was waiting for his n/w package PR to get merged. Need a rebase on that.

Reply:
I checked the current implementation of zaplog, seems like we have to pass the name of the logger file currently to initialize the zaplog.